### ISSUE TITLE
Book marking

### ISSUE THREAD

**Issue by dgcruzing** [MM-Lehmann/pptx2h5p#1]
MM, 

Came through on the H5P Course thread on this.. 

Interested to learn a little more on how to get it up and running. 
I see this is not a window install set and looks like we have to run it through a windows shell.. ?

Love the idea and as a lecturer, this would be a great tool to our kit. 

Any chance of getting it as a standalone win10 install?

**Comment by MM-Lehmann** [comment]
Hi @dgcruzing 
thanks for reaching out. You can find instructions on the release section.
There is no need to install anything. Just copy the file anywhere on your system and place a shortcut into your context menu for easy access.
I might put some better instructions on the README at some point

**Comment by dgcruzing** [comment]
> Hi @dgcruzing
> thanks for reaching out. You can find instructions on the release section.
> There is no need to install anything. Just copy the file anywhere on your system and place a shortcut into your context menu for easy access.
> I might put some better instructions on the README at some point

MM, thanks for that, 
Key bit there I was missing was looking in the release section.. ;-)

Got it working nice.. 

For those a little less techy that come across this.. 

Create a shortcut on your desktop to it. i.e right click on "ppt2h5p"..

Open your explorer, type shell:sendto into the addressbar 

hit enter, this should take you to your "users/yourusername/Appdata/roaming/Microsoft/windows/Sendto

drag the shortcut you have created in to here

As per MM instructions, right-click on your PPT, send to "ppt2h5p". Let the magic happen. 
Upload load into your H5P repository 
Profit. 
buy the man a beer.. :-)

**Comment by dgcruzing** [comment]
MM, Sorry to be pesky... 

Just got to ask if there is some German language connection here. 

As I used it for the lecture today and on conversion, it loaded into my H5P.com site with German language settings?

Touched base with the H5P team and worked out that I can use the language override settings so all good. 

Just wanted to see if it might be coming out of this conversion?

**Comment by MM-Lehmann** [comment]
Yes, you're right. I took a german template file off an instance on a German University Server. Something I would like to investigate further next time I touch the project.

**Comment by MM-Lehmann** [comment]
Yes, you're right. I took a german template file off an instance on a German University Server. Something I would like to investigate further next time I touch the project.

**Comment by MM-Lehmann** [comment]
The `default language` is a required field in the `contents.json`. But upon import, users can change the language setting in the `override` settings below (e.g. on h5p).
I'm going to close this for now until some other ideas come up.

**Comment by paulofvieira** [comment]
Several people had some dificulties, so i made a bat file

@echo off
set /p id=Name of the File: 
echo %id%
pptx2h5p.exe %id%.pptx

0 - Put the files pptx2h5p.exe, transform.bat, and the powerpoint file in same directory
1 - Execute the file transform.bat;
2 - You will be asked for the name of the file to transform (without extension)
3 - Input  the name of the file
4 - Press ENTER
Done

**Comment by MM-Lehmann** [comment]
Thank you for the template, Paolo.
  
As an alternative for the less tech-savyy:
  
Simply drag (and drop) a `*.pptx` file onto the `pptx2h5p.exe`. this becomes easier, when both reside in the same folder, but it's not required.
  
In theory, this should also work with multiple files at the same time.
  
Happy new year!
  
Martin
  

  

  
  
  
  
>

**Comment by nicolawestermann** [comment]
Hallo Martin, ich bin gerade auf Dein interessantes Tool gestoßen! Eine Frage: Erreicht man damit bessere Ergebnisse in Bezug auf die Bildauflösung, als wenn man eine PPT-Folie einfach in png umwandelt und in H5P hochlädt und auf die Größe anpasst? Bei mir werden die so in H5P hochgeladenen und an die Größe angepassten Bilder immer ein bisschen zu unscharf.
Vielen Dank für Deine Hilfe. Kommunikation auch gerne als pn an nicolawestermann@gmail.com. Herzliche Grüße

**Comment by MM-Lehmann** [comment]
Hi Nicola, Because it might be relevant to other users, I will answer here directly.   
  
I believe this tool actually preserves resolution and quality as much as possible (at least there is no intended downscaling) but it might be an interesting test to directly compare the results of this tool against the manual method you described below. Please, let us know if your issue with image quality got resolved by using this tool!
  

  

  
  
  
  
  
>
